### PR TITLE
Edit javascript codebyte1

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- The following terms were updated:

  - JavaScript/Dates: .getDate(),
  - JavaScript/Dates: .getDay(),
  - JavaScript/Dates: .getHours(),

- The following changes were made:
  - Add codebyte example,
  - Fix typos,
  - Update term definitions to match other terms

- Closes: #2226 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getDate/getDate.md [hunk 1]
```diff
@@ -35,3 +37,14 @@ if (today.getDate() < 10) {
 
 // Output: It is the 10th, or later!
 ```
+
+## Codebyte Example
+
+The example below is runnable, and first, it defines a variable `myDate` with a given new `Date`, then uses the `getDate()` method to store the day from that date into a variable called `payDay`. Then `console.log` prints this day to the screen:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDate/getDate.md:43]:
```suggestion
The example below is runnable, and assigns the variable `myDate` to a new given `Date`, then the result of the `.getDate()` method is stored in `payDay`. Finally, a `console.log` call prints this value to the screen:
```

File: content/javascript/concepts/dates/terms/getDate/getDate.md [hunk 2]
```diff
@@ -12,14 +12,16 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Called from an instance of the `Date` class, will return the date of the month. All return values will be integers between `1` and `31`.
+The **`.getDate()`** method returns the day of the month.
 
 ## Syntax
 
 ```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDate/getDate.md:19]:
```suggestion
```shell
```

File: content/javascript/concepts/dates/terms/getDay/getDay.md [hunk 3]
```diff
@@ -12,19 +12,19 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Called from an instance of the `Date` class, will return the day of the week.
+The **`.getDay()`** method returns the day of the week.
 
 ## Syntax
 
 ```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDay/getDay.md:19]:
```suggestion
```shell
```

File: content/javascript/concepts/dates/terms/getDate/getDate.md [hunk 4]
```diff
@@ -12,14 +12,16 @@ CatalogContent:
...
 myDate.getDate();
 ```
 
+The return value of `getDate()` is an integer number between `1` and `31` representing the day of a month.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDate/getDate.md:23]:
```suggestion
The return value of `.getDate()` is an integer number between `1` and `31` representing the day of a month.
```

File: content/javascript/concepts/dates/terms/getDay/getDay.md [hunk 5]
```diff
@@ -12,19 +12,19 @@ CatalogContent:
...
 myDate.getDay();
 ```
 
-The return value of `getDay()` will only be an integer number between `0` and `6`:
+The return value of `getDay()` is an integer number between `0` and `6`:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDay/getDay.md:23]:
```suggestion
The return value of `.getDay()` is an integer number between `0` and `6`:
```

File: content/javascript/concepts/dates/terms/getDay/getDay.md [hunk 6]
```diff
@@ -45,3 +45,14 @@ if (today.getDay() === 5) {
 
 // Output: It is Friday, let's have a coffee later!
 ```
+
+## Codebyte Example
+
+The example below is runnable, and first, it defines a variable called `today` with a new `Date` of today's date, then uses the `getDay()` method to store the weekday from that date into a variable called `weekDay`. Then `console.log` prints the number representing that day of the week to the screen:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getDay/getDay.md:51]:
```suggestion
The example below is runnable, and first, it defines `today` with today's date, then the result of the `.getDay()` method is assigned to `weekDay`. Finally, a `console.log` call prints the value to the screen:
```

File: content/javascript/concepts/dates/terms/getHours/getHours.md [hunk 7]
```diff
@@ -12,15 +12,15 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Called from an instance of the `Date` class, will return the hour according to the local time.
+The **`.getHours()`** method returns the hours of a date according to the local time.
 
 ## Syntax
 
 ```js
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getHours/getHours.md:19]:
```suggestion
```shell
```

File: content/javascript/concepts/dates/terms/getHours/getHours.md [hunk 8]
```diff
@@ -12,15 +12,15 @@ CatalogContent:
...
 myDate.getHours();
 ```
 
-The return value of `getHours()` will only be an integer number between `0` and `23`, meaning the hour according to the local time.
+The return value of `getHours()` is an integer number between `0` and `23`, meaning the hour according to the local time.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getHours/getHours.md:23]:
```suggestion
The return value of `.getHours()` is an integer number between `0` and `23`, meaning the hour according to the local time.
```

File: content/javascript/concepts/dates/terms/getHours/getHours.md [hunk 9]
```diff
@@ -37,3 +37,14 @@ if (today.getHours() === 10) {
 
 // Output: It is not 10 in the morning now.
 ```
+
+## Codebyte Example
+
+The example below is runnable, and first, it defines a variable called `currentTime` with a new `Date` of today's date and time, then uses the `getHours()` method to store the hour from that date into a variable called `currentHour`. Then `console.log` prints the number representing that hour of the day to the screen:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getHours/getHours.md:43]:
```suggestion
The example below is runnable, and first, it defines `currentTime` with today's date and time, then the result of the `.getHours()`method is saved in `currentHour`. Lastly, a `console.log` call prints the value to the screen:
```

File: content/javascript/concepts/dates/terms/getHours/getHours.md [hunk 10]
```diff
@@ -37,3 +37,14 @@ if (today.getHours() === 10) {
...
+
+```codebyte/javascript
+const currentTime = new Date();
+const currentHour = currentTime.getHours();
+
+console.log("The current hour of the day is: " + currentHour);
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getHours/getHours.md:49]:
```suggestion
console.log("The current hour is: " + currentHour);
```

File: content/javascript/concepts/dates/terms/getDate/getDate.md (hunk 1 above)
Comment (caupolicandiaz) [reply content/javascript/concepts/dates/terms/getDate/getDate.md:43]: Generally, I think these example introductions should be minimal.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've completed my initial review. Please update the entries, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've completed my initial review. Please update the entries, and tag me when you've finished making changes. Thanks

**Comment by cslylla** [comment]
@caupolicandiaz I approved the changes.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Looks good, this is ready for a second review.

**Comment by SSwiniarski** [comment]
@yangc95, @KTom101, @caupolicandiaz, @HishamT-- Just noted that this seems to have overwritten the original authors for these entries.

**Comment by KTom101** [comment]
@SSwiniarski In previous entries when the original author was anonymous, the author of the edit is listed and the anonymous author thumbnail wouldn't appear anymore. But I'm not sure if this is what happened here, do we know if the previous authors were anonymous?

**Comment by yangc95** [comment]
@SSwiniarski @KTom101 The original author was @EugeneGohh and it looks like the entries written are anonymous. See: https://www.codecademy.com/resources/docs/javascript/dates/getMinutes which was also written by Eugene.

**Comment by KTom101** [comment]
Thanks @yangc95 ❤️

**Pull request merged**